src/core/bbr.h

```diff
@@ -91,6 +68,16 @@ typedef struct QUIC_CONGESTION_CONTROL_BBR {
     // If TRUE, ProbeRttEndTime is valid
     //
     BOOLEAN ProbeRttEndTimeValid : 1;
+
+    //
+    // If TRUE, current RTT sample is expired
+    //
+    BOOLEAN RttSampleExpired: 1;
```

Comment:
This struct could definitely do with some reordering to fix the huge amount of extra padding in here (perhaps in a follow up PR?).

Reply:
Yeah let's do this at the end of the cleanup pass in case fields are added/removed